tests/PySys/tedge_mapper_c8y/runtime_start_two_mappers/run.py

```diff
@@ -20,22 +19,26 @@ def execute(self):
         tedge_mapper = "/usr/bin/tedge_mapper"
         self.sudo = "/usr/bin/sudo"
 
-        tedge_mapper1 = self.startProcess(
+        # starting the first tedge mapper instance
+        _ = self.startProcess(
             command=self.sudo,
             arguments=["systemctl", "start", "tedge-mapper-c8y"],
             stdouterr="tedge_mapper1",
         )
 
         self.wait(0.1)
 
-        tedge_mapper2 = self.startProcess(
+        # starting the second tedge mapper instance
+        # and expecting this one to crash
+        _ = self.startProcess(
             command=self.sudo,
-            arguments=["-u", "tedge-mapper", tedge_mapper, "c8y"],
+            arguments=["-u", "tedge", tedge_mapper, "c8y"],
```

Comment:
@abelikt this is the change i made. I tested locally and it works. I ran: 

```pysys.py run --record -v DEBUG 'runtime_start_two_mapper*'```

**But this also worked before the change, so please can you test on your side too?**